packages/web/src/scss/components/Icon/_Icon.scss

```diff
@@ -1,7 +1,3 @@
-@use 'theme';
-
 .Icon {
     display: inline-block;
```

Comment:
Is this necessary at all? Is there any difference?

Reply:
TODO consider removing Icon styles at all. As we set up a size on SVG element directly we might not need this.

Reply:
Please remove the CSS in this PR and then I'll see if it's convenient to move all icons elsewhere.